parent/pom.xml

```diff
@@ -44,7 +46,7 @@
     <version.commons-logging>1.2</version.commons-logging>
     <version.commons-math>3.6.1</version.commons-math>
     <version.commons-codec>1.14</version.commons-codec>
-    <version.elasticsearch>6.8.6</version.elasticsearch>
+    <version.elasticsearch>6.8.9</version.elasticsearch>
```

Comment:
Is this required? I think it is good not to change the versions when backporting.

Reply:
Required - yes and no

There was a security issue with the log4j library. On bumping the log4j version, then the Elasticsearch version had a dependency version conflict, so I bumped it, too.

I agree with your sentiment that changing versions during backport is not trivial. My personal rule of thumb is that I try to limit it to patch version changes only.

If you insist I can roll it back. In this case I would open another issue to bump the log4j version.

Please advise.